source/_components/media_player.nadtelnet.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "NAD"
+description: "Instructions how to integrate NAD receivers into Home Assistant."
+date: 2018-01-16 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nad.png
+ha_category: Media Player
+ha_release: 0.62
+ha_iot_class: "Local Polling"
+---
+
+
+The `nadtelnet` platform allows you to control a [NAD receiver](http://nadelectronics.com) through telnet from Home Assistant.
+
+To add an NAD receiver to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: nadtelnet
+    host: 192.168.0.150
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration